packages/react/src/components/Pagination/Pagination.mdx

```diff
@@ -23,49 +23,48 @@ Vær oppmerksom på:
 
 ### Eksempel på bruk med `Pagination`
 
-`Pagination` er en kontrollert komponent. Det vil si at komponentens tilstand om hvilken side som er aktiv styres utenfra. Når komponenten er på første eller siste side, skjules "forrige"/"neste"-pilene.
+`Pagination` er en kontrollert komponent. Det vil si at komponentens tilstand om hvilken side som er aktiv styres utenfra.
 
 ```tsx
 import { Pagination } from '@digdir/designsystemet-react';
 
-const [currentPage, setCurrentPage] = useState(1);
+const [currentPage, setCurrentPage] = useState(4);
+const { pages, goNext, goPrevious, hasNext, hasPrevious } = usePagination({
+  currentPage,
+  setCurrentPage,
+  totalPages: 10,
+  showPages: 7,
+});
 
-<Pagination
-  currentPage={1}
-  totalPages={10}
-  onChange={setCurrentPage}
-  nextLabel='Neste'
-  previousLabel='Forrige'
-  itemLabel={(num) => `Side ${num}`}
-/>;
-```
-
-### Eksempel på bruk med del-komponenter
-
-Er det behov for en tilpasset `Pagination` kan du ta i bruk de enkelte del-kompontenene.
-
-```tsx
-<Pagination.Root>
+<Pagination>
   <Pagination.List>
     <Pagination.Item>
-      <Pagination.Previous>
-        <ChevronLeftIcon aria-hidden />
+      <Pagination.Button
+        aria-label='Forrige side'
+        disabled={!hasPrevious}
+        onClick={goPrevious}
+      >
         Forrige
-      </Pagination.Previous>
+      </Pagination.Button>
     </Pagination.Item>
-
-    <Pagination.Item>
-      <Pagination.Button isActive>1</Pagination.Button>
-    </Pagination.Item>
-
+    {pages.map(({ page, itemKey, buttonProps }) => (
```

Comment:
Should we rater call it `items`? as this aligns with `Pagination.Item`? :)

Reply:
I'm not sure, not all items are in this, since prev/next buttons are outside the map.. I think pages is fine here personally